### DISCUSSION TITLE
Material Design 3

### DISCUSSION THREAD

**Discussion by WCN-llc** [quasarframework/quasar#17110] [Ideas / Proposals]
Hello, dear colleagues!
Are there any plans to implement Material Design 3 in Quasar?

Thank you!

**Comment by little3201** [comment]
expect

**Comment by ssuess** [comment]
also very curious! Or if there is a third part plugin/ext/adaptation somewhere

**Comment by WCN-llc** [comment]
In the latest version (2.16.8), the Quasar team has significantly extended Sass/SCSS Variables.
I haven't tried to customize with variables on MD3 yet, but there are more possibilities.
[https://quasar.dev/style/sass-scss-variables](url)

**Reply by karelbecerra** [reply]
tiny change but could you please fix link (text and link do not match)

**Comment by anoyomoose** [comment]
I've given it an attempt here - https://github.com/quasarframework/quasar/discussions/18269

By no means finished but it's some of the way there :)
